frontend/packages/console-shared/src/components/pod/PodStatus.tsx

```diff
@@ -24,6 +24,7 @@ type PodStatusProps = {
   data: ExtPodKind[];
   showTooltip?: boolean;
   title?: string;
+  titleComponent?: React.ReactElement<any>;
```

Comment:
A type is not defined.

```suggestion
  titleComponent?: React.ReactElement;
```